src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -658,11 +658,18 @@
   <data name="InvalidLinkDialog.PrimaryButtonText" xml:space="preserve">
     <value>Cancel</value>
   </data>
+  <data name="UnopenedUriText" xml:space="preserve">
+    <value>{}</value>
+    <comment> {} will be replaced with the URI we could not open or parse </comment>
+  </data>

```

Comment:
Ah! We don’t need a localized string just for a format placeholder. We don’t need to use `fmt` at all! (Just pop the URI string directly into the text Run)

Reply:
Omg of course. Thanks for catching that!